CMakeLists.txt

```diff
@@ -126,16 +129,16 @@ include_directories(
 # add_dependencies(iarc7_motion ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
 ## Declare a C++ executable
-# add_executable(iarc7_motion_node src/iarc7_motion_node.cpp)
+add_executable(LowLevelMotionController src/LowLevelMotionController.cpp src/FeedForwardPid.cpp src/QuadVelocityController.cpp)
```

Comment:
I've always seen CMake targets as lowercase_with_underscores in ROS projects, so I assumed that was the convention we were using, but we don't have to